<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{background-image: url(:/res/robot.png);  background-repeat: no-repeat; /* 禁止平铺 */
    background-position: center; /* 图片居中 */
    background-size: cover; /* 按比例缩放，完整显示图片 */}
QLabel {color: rgb(255, 255, 255);}
QLineEdit {
    background-color: rgba(255, 255, 255, 0.2); /* 白色半透明（20%不透明度） */
    border: 1px solid rgba(255, 255, 255, 0.3); /* 边框半透明 */
    color: white; /* 文字白色（避免透明背景下看不清） */
    padding: 5px; /* 内边距 */
}

/* 3. 列表/文本区域（歌曲列表、歌词、历史搜索）透明 */
QListWidget, QPlainTextEdit, QScrollArea {
    background-color: rgba(0, 0, 0, 0.1); /* 黑色半透明（10%不透明度） */
    border: 1px solid rgba(255, 255, 255, 0.2); /* 边框轻微可见 */
    color: white; /* 文字白色 */
}
QToolButton{
	background-color: rgba(255, 255, 255,15);
 border: 1px solid rgba(255, 255, 255, 0.2); /* 边框轻微可见 */
}
/* 4. 按钮（搜索按钮、播放控制按钮）透明 */
QPushButton {
    background-color: rgba(255, 255, 255, 0.15); /* 白色半透明 */
    border: none; /* 无边框（或轻微边框） */
    color: white; /* 文字白色 */
    padding: 5px 10px;
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.3); /*  hover时稍亮 */
}

/* 5. 滑块（进度条）透明 */
QSlider::groove:horizontal {
    background-color: rgba(255, 255, 255, 0.2);
    height: 4px;
    border-radius: 2px;
}
QSlider::handle:horizontal {
    background-color: white;
    width: 12px;
    margin: -4px 0; /* 扩展滑块点击区域 */
    border-radius: 6px;
}

QSpliter{background-color: rgba(0, 0, 0, 5);}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,0,0">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditClear"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::ClickFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:{rgb(85, 255, 255)}
background-color: rgb(85, 170, 0);</string>
           </property>
           <property name="text">
            <string>歌曲名称：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="search_box">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="search_btn">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:{rgb(85, 255, 255)}
background-color: rgb(85, 170, 0);</string>
           </property>
           <property name="text">
            <string>搜索</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="2,1">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>歌曲</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="music_list"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>歌词</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="lyric"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>历史搜索</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="history"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QSlider" name="processbar">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="timer">
              <property name="frameShape">
               <enum>QFrame::Shape::Box</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Sunken</enum>
              </property>
              <property name="text">
               <string>00：00</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pre">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pause">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="QIcon::ThemeIcon::MediaPlaybackPause"/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="next">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset theme="QIcon::ThemeIcon::MediaSkipForward"/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QSlider" name="volume">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="value">
                 <number>50</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TickPosition::NoTicks</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="volume_value">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>50</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
